presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaClientSecurityProtocol.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import java.util.Locale;
+
+public enum KafkaClientSecurityProtocol
+{
+    PLAINTEXT(0, "PLAINTEXT"),
+    SSL(1, "SSL");
+
+    public final short id;
```

Comment:
enum has ordinal field, no need for dedicated `id`

Reply:
done